soh/soh/Enhancements/debugger/actorViewer.cpp

```diff
@@ -918,122 +887,104 @@ void ActorViewerWindow::DrawElement() {
+                        ImGui::InputScalar("X##CurPos", ImGuiDataType_Float, &display->world.pos.x);
+                        ImGui::InputScalar("Y##CurPos", ImGuiDataType_Float, &display->world.pos.y);
+                        ImGui::InputScalar("Z##CurPos", ImGuiDataType_Float, &display->world.pos.z);
+                        ImGui::PopItemWidth();
+                        PopStyleInput();
+                    },
+                    "Actor Position");
+                ImGui::SameLine();
+                DrawGroupWithBorder(
+                    [&]() {
+                        PushStyleInput(THEME_COLOR);
+                        ImGui::PushItemWidth(ImGui::GetFontSize() * 6);
+                        ImGui::Text("Actor Rotation");
+                        ImGui::InputScalar("X##CurRot", ImGuiDataType_S16, &display->world.rot.x);
+                        ImGui::InputScalar("Y##CurRot", ImGuiDataType_S16, &display->world.rot.y);
+                        ImGui::InputScalar("Z##CurRot", ImGuiDataType_S16, &display->world.rot.z);
+                        ImGui::PopItemWidth();
+                        PopStyleInput();
+                    },
+                    "Actor Rotation");
+
+                if (display->category == ACTORCAT_BOSS || display->category == ACTORCAT_ENEMY) {
                     PushStyleInput(THEME_COLOR);
-                    ImGui::PushItemWidth(ImGui::GetFontSize() * 6);
-                    ImGui::Text("Actor Rotation");
-                    ImGui::InputScalar("X##CurRot", ImGuiDataType_S16, &display->world.rot.x);
-                    ImGui::InputScalar("Y##CurRot", ImGuiDataType_S16, &display->world.rot.y);
-                    ImGui::InputScalar("Z##CurRot", ImGuiDataType_S16, &display->world.rot.z);
-                    ImGui::PopItemWidth();
+                    ImGui::InputScalar("Enemy Health", ImGuiDataType_U8, &display->colChkInfo.health);
                     PopStyleInput();
-                },
-                "Actor Rotation");
-
-            if (display->category == ACTORCAT_BOSS || display->category == ACTORCAT_ENEMY) {
-                PushStyleInput(THEME_COLOR);
-                ImGui::InputScalar("Enemy Health", ImGuiDataType_U8, &display->colChkInfo.health);
-                PopStyleInput();
-                UIWidgets::InsertHelpHoverText("Some actors might not use this!");
-            }
-
-            DrawGroupWithBorder(
-                [&]() {
-                    ImGui::Text("flags");
-                    UIWidgets::DrawFlagArray32("flags", display->flags);
-                },
-                "flags");
-
-            ImGui::SameLine();
-
-            DrawGroupWithBorder(
-                [&]() {
-                    ImGui::Text("bgCheckFlags");
-                    UIWidgets::DrawFlagArray16("bgCheckFlags", display->bgCheckFlags);
-                },
-                "bgCheckFlags");
-
-            if (Button("Refresh", ButtonOptions().Color(THEME_COLOR))) {
```

Comment:
Calling this out since the whitespace changes hide this, but I removed this button. It didn't do anything useful than refresh the actor list, which is done a different way now. This also removes the need of the retrieval method.